agent/caddy/Dockerfile

```diff
@@ -0,0 +1,4 @@
+# note: never use the :latest tag in a production site
+FROM caddy
+
+COPY Caddyfile /etc/caddy/Caddyfile
```

Comment:
Any special reason not to use a volume mount in `docker-compose.yaml` for this?

edit: might be a WIP since I do not see the Caddy service defined in  docker-compose

Reply:
Just to avoid having to declare this mounted volume. Ideally users should point the server to an acapy deployed, doesn't have to be the configurations from this repo, they are used as examples. I personally deploy on railway and I just point my service to this dockerfile without having to mount a volume. This simplifies deployment significantly.